manifests/apache.pp

```diff
@@ -54,7 +54,6 @@
   Stdlib::Absolutepath $pki_dir = $certs::pki_dir,
   Optional[Stdlib::Absolutepath] $server_cert = $certs::server_cert,
   Optional[Stdlib::Absolutepath] $server_key = $certs::server_key,
-  Optional[Stdlib::Absolutepath] $server_cert_req = $certs::server_cert_req,
```

Comment:
Why was that ever needed?  (curious-Evgeni is afraid  of the answer)

Reply:
I am not sure -- https://github.com/theforeman/puppet-certs/blob/39713ae873a7865d367ecbbc39a981e3c45d46a0/lib/puppet/provider/cert/katello_ssl_tool.rb#L20 seems symbolic